paragraph/scheduling/graph_scheduler.cc

```diff
@@ -108,6 +123,9 @@ SubroutineFsm& GraphScheduler::GetFsm(const Subroutine* subroutine) {
   return subroutine_state_map_.at(subroutine);
 }
 
+double GraphScheduler::GetCurrentTime() { return current_time_; }
```

Comment:
Who/what calls this?

Reply:
it is used in InstructionFsm when it assigns time_ready or calls scheduler->InstructionFinished for Null and non-leaf instructions